seleniumByTechnocredits/src/seleniumPractise/saket/Assignment4.java

```diff
@@ -0,0 +1,49 @@
+package seleniumPractise.saket;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+
+public class Assignment4 {
+
+	public static void main(String[] args) throws InterruptedException {
+		// TODO Auto-generated method stub
+
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/#");
+		Thread.sleep(1000);
+
+		driver.manage().window().maximize();
+		driver.findElement(By.id("basicelements")).click();
+		Thread.sleep(2000);
+		WebElement element = driver.findElement(By.xpath("//button[@id='javascriptPromp']"));
+		JavascriptExecutor js = (JavascriptExecutor) driver;
+		js.executeScript("arguments[0].scrollIntoView(true)", element);
+ 
+		element.click();
+		Alert alert = driver.switchTo().alert();
+		Thread.sleep(2000);
+		System.out.println("I am in alert");
+		String name="saket";
+		alert.sendKeys(name);
+		System.out.println("I have send the key in alert");
+		alert.accept();
+		
+		String s="Hello saket! How are you today?";
```

Comment:
Instead of hardcoded string you can use 
	String txtok = driver.findElement(By.xpath("//p[@class='help-block' and @id='pgraphdemo']")).getText();

so even though the text changes tomorrow you dont have to pdate your code

Reply:
agreed @AmarjaBTechnocredits done with changes in the hardcoded text